src/ClientBuilder.php

```diff
@@ -41,18 +41,12 @@
  * @method setSampleRate(float $sampleRate)
  * @method string getMbDetectOrder()
  * @method setMbDetectOrder(string $detectOrder)
- * @method bool getAutoLogStacks()
- * @method setAutoLogStacks(bool $enable)
+ * @method bool getAttachStacktrace()
+ * @method setAttachStacktrace(bool $enable)
  * @method int getContextLines()
  * @method setContextLines(int $contextLines)
- * @method string getCurrentEnvironment()
- * @method setCurrentEnvironment(string $environment)
- * @method string[] getEnvironments()
- * @method setEnvironments(string[] $environments)
- * @method string[] getExcludedLoggers()
- * @method setExcludedLoggers(string[] $loggers)
- * @method string[] getExcludedExceptions()
- * @method setExcludedExceptions(string[] $exceptions)
+ * @method null|string getEnvironment()
+ * @method setEnvironment(?string $environment)
```

Comment:
The PHPDoc standard doesn't allow `nullable` values to be defined with the syntax PHP uses (the exclamation mark before the type). You always have to use the union of the type and the `null` value